src/mono/wasm/build/WasmApp.targets

```diff
@@ -255,6 +256,7 @@
     <PropertyGroup>
       <EmccFlags>$(_DefaultEmccFlags) $(EmccFlags)</EmccFlags>
       <EmccFlags>$(EmccFlags) -s DISABLE_EXCEPTION_CATCHING=0</EmccFlags>
+      <EmccFlags Condition="'$(WasmNativeStrip)' == 'false'">$(EmccFlags) -g</EmccFlags>
```

Comment:
We have a `$(WasmNativeDebugSymbols)` property that adds the `-g`. Maybe we should set that to true with the condition here?

Reply:
lets do that when me move the options around, I'd like the symbols in asap

Reply:
I've been converting the `-g` to `-g4` for WASM. I've been doing this since it seems that Emscripten requires `-g4` in order to generated source maps. See https://emscripten.org/docs/tools_reference/emcc.html.